packages/ui/src/council/components/election/pastElection/PastElectionStats.tsx

```diff
@@ -21,7 +21,7 @@ export const PastElectionStats = ({
   totalVotes,
 }: PastElectionStatsProps) => (
   <Statistics>
-    <StatisticItem title="Ended at">
+    <StatisticItem title="Started at">
       {finishedAtBlock ? formatDateString(finishedAtBlock.timestamp) : '-'}
```

Comment:
Either the finished timestamp should be replaced by the started one, or the copy should stay the same. Also IMO it would be nicer to show a block component (I don't remember whether their is a block statistic component...) so their would be direct link to the polkadot app here too.

Reply:
Fixed by e52dd8f